### PR TITLE
Relaxes analytics permissions for admins

### PR DESCRIPTION
**Pull request by sharadmv**

### CODE REVIEW

File: server/app/analytics/job.py [hunk 1]
```diff
@@ -135,9 +135,13 @@ def update_status(self, status):
...
     def _can(cls, user, need, obj, query):
         if need.action == "index":
+            if user.is_admin:
+                return query
             return query.filter(AnalyticsDump.owner == user.key)
         if need.action == "create":
             return True
         if need.action == "get":
+            if user.is_admin:
+                return True
```
Comment (Some3141) [inline server/app/analytics/job.py]: Can we just say at the top "if user.is_admin: return True"?

Comment (sharadmv) [inline server/app/analytics/job.py]: No, we need to handle the index case separately since we return `query`.

Comment (Some3141) [inline server/app/analytics/job.py]: Oh, didn't see that.

**Comment by Some3141** [comment]
LGTM.

**Pull request merged**